BlocksForAll/SetVariableModViewController.swift

```diff
@@ -9,12 +9,28 @@
 import Foundation
 import UIKit
 
-class SetVariableModViewController: UIViewController, UITextFieldDelegate{
+
+class SetVariableModViewController: UIViewController {
     
     var modifierBlockIndexSender: Int?
     var variableSelected: String = "orange"
     var variableValue: Int = 0
+
     
```

Comment:
lines 20 to 33
@IBOutlet weak var orangeButton: UIButton!
     @IBOutlet weak var bananaButton: UIButton!
     @IBOutlet weak var cherryButton: UIButton!
     @IBOutlet weak var watermelonButton: UIButton!
     @IBOutlet weak var appleButton: UIButton!

 
      func deselectAll(){
         orangeButton.layer.borderWidth = 0
         bananaButton.layer.borderWidth = 0
         cherryButton.layer.borderWidth = 0
         watermelonButton.layer.borderWidth = 0
         appleButton.layer.borderWidth = 0
     }